src/Microsoft.ML.DataView/DataViewType.cs

```diff
@@ -150,7 +150,7 @@ public static NumberDataViewType UInt16
             get
             {
                 return _instUInt16 ??
-                    Interlocked.CompareExchange(ref _instUInt16, new NumberDataViewType(typeof(ushort), "U2"), null) ??
+                    Interlocked.CompareExchange(ref _instUInt16, new NumberDataViewType(typeof(ushort), "UIUnt16"), null) ??
```

Comment:
>UIUnt16 [](start = 105, length = 7)

Type.

Reply:
Note that if you had, say, done `nameof(UInt16)` you wouldn't have had this problem. I think in fact this would probably be a good idea to adopt...

---
In reply to: [268029020](https://github.com/dotnet/machinelearning/pull/3046#discussion_r268029020) [](ancestors = 268029020)

Reply:
I think we should just get rid of this string and use `RawType.Name` instead.